### ISSUE TITLE
Installing Chromium without converting it to Chrome

### ISSUE THREAD

**Issue by optio50** [MuntashirAkon/Chrome-OS-Multiboot#5]
In the instructions you say we dont need to do steps 2 & 5 if we are not converting to Chrome.
Thats still leaves step 3 "SWTPM.tar" with no clear indication of what to do with it.

I only ask because I tried the install_chromium.sh many times unsuccessfully.
Script reported a successful install.
It gets to the Chromium boot logo screen and hangs.
I finally tried the conversions script and it worked great.

So.... How do I just do a plain Arnold the bat (Chromium) install? 
I have tried multiple versions of arnoldthebat builds.
They boot fine on USB but I cant get it to work with your script without chromfy (Chrom OS conversion).
Your script seems to be doing something or not doing something (I cant narrow it down)

**Comment by MuntashirAkon** [comment]
> In the instructions you say we dont need to do steps 2 & 5 if we are not converting to Chrome.
Thats still leaves step 3 "SWTPM.tar" with no clear indication of what to do with it.

My bad. Step 3 is also optional.

> So.... How do I just do a plain Arnold the bat (Chromium) install?

To be honest, I've never tried to install vanilla @ArnoldTheBat's builds myself (I created this script for myself, you know). I included this because the process should be similar for vanilla Chromium builds as well. Since people are *actually* using this script, I'll definitely take a look (I'm on vacation right now, I'll take a look at it as soon as I'm back).
